modules/swagger-jaxrs/src/test/scala/HiddenParametersScannerTest.scala

```diff
@@ -0,0 +1,47 @@
+import io.swagger.jaxrs.Reader
+import io.swagger.jaxrs.config.DefaultReaderConfig
+import io.swagger.models.parameters.{BodyParameter, PathParameter, QueryParameter, SerializableParameter}
+import io.swagger.models.properties.{MapProperty, _}
+import io.swagger.models.{ArrayModel, Model, ModelImpl, RefModel, Swagger}
+import models.TestEnum
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FlatSpec, Matchers}
+import resources._
+
+import scala.collection.JavaConverters._
+
+@RunWith(classOf[JUnitRunner])
+class HiddenParametersScannerTest extends FlatSpec with Matchers {
```

Comment:
We are currently migrating tests away from scala - if you could consider rewriting this in pure java (with TestNG) as in https://github.com/swagger-api/swagger-core/blob/develop_2.0/modules/swagger-core/src/test/java/io/swagger/BeanValidatorTest.java that would be much appreciated... thank you!